guides/common/modules/proc_configuring-pulpcore-api-workers.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Configuring_Pulpcore_API-Workers_{context}"]
+= Configuring Pulpcore API Workers
+
+The pulpcore-api workers responds to incoming API requests. Less api workers consume less memory and results in better performance.
+
+For example, we have compared these two setups:
+
+[width="100%",cols="50%,50%",options="header",]
+|===
+|{Project} VM with 8 CPUs, 32 GiB RAM, 3 pulpcore-api_workers |{Project} VM with 8 CPUs, 32 GiB RAM, 10 pulpcore-api_workers
+|===
+
+Outcome looks similar in both cases, so the recommendation would be to lower the number of api workers  because of the improvement in memory consumption.
```

Comment:
```suggestion
Outcome looks similar in both cases, so the recommendation would be to lower the number of API workers because of the improvement in memory consumption.
```